src/libraries/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -1500,6 +1500,14 @@ public static void ToString_InvalidFormat_ThrowsFormatException()
             Assert.Throws<FormatException>(() => f.ToString("E" + intMaxPlus1String));
         }
 
+        [Theory]
+        [InlineData("3.00")]
```

Comment:
I assume this is one of the tests we would expect to add a much longer list of values to in the future, per our recent discussion about adding much more comprehensive tests in this area.

Reply:
Yup. This is just the start